File: app/src/main/java/com/lumiyaviewer/lumiya/slproto/assets/AssetFormatException.java
package com.lumiyaviewer.lumiya.slproto.assets;

public class AssetFormatException extends Exception {
    private static final long serialVersionUID = -8391424207465457690L;

    public AssetFormatException() {
        super("Unsupported asset format");
    }

    public AssetFormatException(String str) {
        super(str);
    }

    public AssetFormatException(String str, Throwable th) {
        super(str, th);
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/slproto/assets/AssetFormatException.kt
package com.lumiyaviewer.lumiya.slproto.assets

open class AssetFormatException : Exception {
    constructor() : super("Unsupported asset format")
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    companion object { private const val serialVersionUID: Long = -8391424207465457690L }
}
